Support logaback 1.2.x

per #144, additonal item discovery

```
2017-02-26 22:35:16,811[ost-startStop-1]  [ERROR] [o.a.c.startup.ContextConfig 1624]    - Failed to detect ServletContainerInitializers for context with name [/docs]
java.io.IOException: java.lang.ClassNotFoundException: ch.qos.logback.classic.servlet.LogbackServletContainerInitializer
	at org.apache.catalina.startup.WebappServiceLoader.loadServices(WebappServiceLoader.java:192)
	at org.apache.catalina.startup.WebappServiceLoader.load(WebappServiceLoader.java:159)
	at org.apache.catalina.startup.ContextConfig.processServletContainerInitializers(ContextConfig.java:1622)
	at org.apache.catalina.startup.ContextConfig.webConfig(ContextConfig.java:1135)
	at org.apache.catalina.startup.ContextConfig.configureStart(ContextConfig.java:775)
	at org.apache.catalina.startup.ContextConfig.lifecycleEvent(ContextConfig.java:299)
	at org.apache.catalina.util.LifecycleBase.fireLifecycleEvent(LifecycleBase.java:94)
	at org.apache.catalina.core.StandardContext.startInternal(StandardContext.java:5087)
	at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:150)
	at org.apache.catalina.core.ContainerBase.addChildInternal(ContainerBase.java:752)
	at org.apache.catalina.core.ContainerBase.addChild(ContainerBase.java:728)
	at org.apache.catalina.core.StandardHost.addChild(StandardHost.java:734)
	at org.apache.catalina.startup.HostConfig.deployDirectory(HostConfig.java:1107)
	at org.apache.catalina.startup.HostConfig$DeployDirectory.run(HostConfig.java:1841)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
Caused by: java.lang.ClassNotFoundException: ch.qos.logback.classic.servlet.LogbackServletContainerInitializer
	at org.apache.catalina.loader.WebappClassLoaderBase.loadClass(WebappClassLoaderBase.java:1285)
	at org.apache.catalina.loader.WebappClassLoaderBase.loadClass(WebappClassLoaderBase.java:1119)
	at java.lang.Class.forName0(Native Method)
	at java.lang.Class.forName(Class.java:348)
	at org.apache.catalina.startup.WebappServiceLoader.loadServices(WebappServiceLoader.java:188)
	... 18 common frames omitted
```

--- a/tomcat7/pom.xml
+++ b/tomcat7/pom.xml
@@ -78,6 +78,14 @@
                                         <exclude>META-INF/services/**</exclude>
                                     </excludes>
                                 </filter>
+
+                                <!-- Exclude services from logback-classic as we redefine them -->
+                                <filter>
+                                    <artifact>ch.qos.logback:logback-classic</artifact>
+                                    <excludes>
+                                        <exclude>META-INF/services/**</exclude>
+                                    </excludes>
+                                </filter>
                             </filters>
 
                             <relocations>

--- a/tomcat8/pom.xml
+++ b/tomcat8/pom.xml
@@ -78,6 +78,14 @@
                                         <exclude>META-INF/services/**</exclude>
                                     </excludes>
                                 </filter>
+
+                                <!-- Exclude services from logback-classic as we redefine them -->
+                                <filter>
+                                    <artifact>ch.qos.logback:logback-classic</artifact>
+                                    <excludes>
+                                        <exclude>META-INF/services/**</exclude>
+                                    </excludes>
+                                </filter>
                             </filters>
 
                             <relocations>

--- a/tomcat85/pom.xml
+++ b/tomcat85/pom.xml
@@ -78,6 +78,14 @@
                                         <exclude>META-INF/services/**</exclude>
                                     </excludes>
                                 </filter>
+
+                                <!-- Exclude services from logback-classic as we redefine them -->
+                                <filter>
+                                    <artifact>ch.qos.logback:logback-classic</artifact>
+                                    <excludes>
+                                        <exclude>META-INF/services/**</exclude>
+                                    </excludes>
+                                </filter>
                             </filters>
 
                             <relocations>

--- a/tomcat9/pom.xml
+++ b/tomcat9/pom.xml
@@ -78,6 +78,14 @@
                                         <exclude>META-INF/services/**</exclude>
                                     </excludes>
                                 </filter>
+
+                                <!-- Exclude services from logback-classic as we redefine them -->
+                                <filter>
+                                    <artifact>ch.qos.logback:logback-classic</artifact>
+                                    <excludes>
+                                        <exclude>META-INF/services/**</exclude>
+                                    </excludes>
+                                </filter>
                             </filters>
 
                             <relocations>

